.github/workflows/kvrocks.yaml

```
@@ -50,6 +50,16 @@ jobs:
         run: |
           echo "docs_only=${{ fromJSON(steps.changes.outputs.all_count) == fromJSON(steps.changes.outputs.docs_count) && fromJSON(steps.changes.outputs.docs_count) > 0 }}" >> $GITHUB_OUTPUT
 
+  check-typos:
+    name: Check typos
+    runs-on: ubuntu-22.04
+    env:
+      FORCE_COLOR: 1
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run typos check
+        uses: crate-ci/typos@master
```

The configuration file can be specified
https://github.com/crate-ci/typos/blob/ef5fcf92dfbd679f97c0371159e143852f7b1eb1/action.yml#L28

```suggestion
      - name: Run typos check
        uses: crate-ci/typos@master
        with:
          config: .github/config/typos.toml
```

---

Thank you!